### PR TITLE
Merge for web app changes

### PR DESCRIPTION
**Pull request by poojaRal**
**Taiga User Story**
Tg-597
**Taiga Task(s)**
Tg-654
**What did you change?**
Added a route for the web-application to be hosted from the node server using the path of the web-app  mentioned in config.json
**How can we test?**
1. run npm init
2. Check that config.json file is created.
3. Edit the config.json file and add the below after "database" object
``` json
"application": {
     "path": "<path pointing to the web-app part on the server"
 }
```
4. run npm start
5. Check that the node server is hosted successfully.
6. Check for the route "/promis" and see that the web-application is hosted.

### CODE REVIEW

File: controller/server.js [hunk 1]
```diff
@@ -118,6 +118,20 @@ function dashboardServer (configuration) {
 
     // load application routes
     server.route(router);
+    if (configuration.application) {
```
Comment (ChristianMurphy) [inline controller/server.js]: maybe add a comment `// optionally add survey application routes`

**Comment by ChristianMurphy** [comment]
:shipit:

File: controller/server.js (hunk 1 above)
Comment (poojaRal) [inline controller/server.js]: Added a comment

Comment (ChristianMurphy) [inline controller/server.js]: :sparkles:

**Comment by iamaureen** [comment]
LGTM

**Pull request merged**